Transactions.md

```diff
@@ -169,3 +169,45 @@ Where
 
 Where
   - `P2WPKH(Local Penalty Public Key)`'s value is `P2WSH(Remote's ToLocalOutput).value - fee`
+
+## Mutual Closing Transaction
+### <a name="MutualClosingKnownValues">Known Values</a>
+  * Local Mutual Closing Public Key: `ECPublicKey`
+  * Local Payout: `CurrencyUnit`
+  * Remote Mutual Closing Public Key: `ECPublicKey`
```

Comment:
Doesn't have to happen on this PR but I think that KeyDerivation.md needs an update

Reply:
I went ahead and did it: https://github.com/discreetlogcontracts/dlcspecs/pull/7

Reply:
OK, Thanks.